dlog/kimchi/src/prover.rs

```diff
@@ -487,13 +489,23 @@ where
         t += &bnd;
 
         // commit to t
-        let t_comm = index.srs.commit(&t, None, rng);
+        let t_comm = {
+            let (mut t_comm, mut omega_t) = index.srs.commit(&t, None, rng);
+
+            let expected_t_size = 7;
+            let dummies = expected_t_size - t_comm.unshifted.len();
+            // Add some hiding commitments to 0
+            for _ in 0..dummies {
+                use ark_ec::ProjectiveCurve;
+                let w = Fr::<G>::rand(rng);
+                t_comm.unshifted.push(index.srs.h.mul(w).into_affine());
+                omega_t.unshifted.push(w);
+            }
+            (t_comm, omega_t)
```

Comment:
can you explain that change? The comment seems to imply that the padding can only exist when the commitment is to 0

Reply:
Yes, the padding is some commitments to 0, since we only have to add padding when the upper degree coefficients of t are 0.